scholia/app/templates/chemical-class_related-chemicals.sparql

```diff
@@ -2,7 +2,8 @@ PREFIX target: <http://www.wikidata.org/entity/{{ q }}>
 
 SELECT ?mol ?molLabel ?InChIKey ?CAS ?CASUrl ?ChemSpider ?ChemSpiderUrl ?PubChem_CID ?PubChem_CIDUrl WITH {
 SELECT DISTINCT ?mol WHERE {
-  ?mol wdt:P31/wdt:P279* target: .
+  ?mol wdt:P279* target: .
+  MINUS { [] wdt:P279 ?mol }
```

Comment:
Could these two lines not be changed to `?mol wdt:P279+ target:`?

Reply:
The `MINUS` is in the other direction.

The `?mol wdt:P279* target: .` could be `?mol wdt:P279+ target: .`, but do you think that makes a difference? What would it match for no `wdt:P279` ? (I do not actually know...)

Reply:
Will `?mol wdt:P279* target:` not match target=?mol ?

Reply:
I'll check.

Reply:
Isn't MINUS there to avoid that `?mol` becomes `target:`?

Reply:
No. The MINUS is to ensure it only colors things orange that are not a superclass themselves.